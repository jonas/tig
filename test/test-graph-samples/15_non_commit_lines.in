commit A B C
    Commit A - merge B and C
commit C B
    Commit C - after B
commit B D E
    Commit B - merge D and E
commit D E
    Commit D - after E
commit E F
    Commit E - after F
commit F G H I J
    Commit F - merge G, H, I, J
commit H G
    Commit H - after G
commit I G
    Commit I - after G
commit G K
    Commit G - after K
commit J L M
    Commit J - merge L and M
commit K L
    Commit K - after L
commit M L
    Commit M - after L
commit L
    Commit L
