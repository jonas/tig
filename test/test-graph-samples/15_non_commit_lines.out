●─╮ Commit A - merge B and C
│ ● Commit C - after B
├─╯
●─╮ Commit B - merge D and E
● │ Commit D - after E
●─╯ Commit E - after F
●─┬─┬─╮ Commit F - merge G, H, I, J
│ ● │ │ Commit H - after G
│ │ ● │ Commit I - after G
●─┴─╯ │ Commit G - after K
│ ╭───╯
│ ●─╮ Commit J - merge L and M
● │ │ Commit K - after L
│ │ ● Commit M - after L
◎─┴─╯ Commit L
